sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -35,3 +35,42 @@ case class Size(child: Expression) extends UnaryExpression with ExpectsInputType
     nullSafeCodeGen(ctx, ev, c => s"${ev.primitive} = ($c).size();")
   }
 }
+
+/**
+ * Sorts the input array in ascending order according to the natural ordering of
+ * the array elements and returns it.
+ */
+case class SortArray(child: Expression)
+  extends UnaryExpression with ExpectsInputTypes with CodegenFallback {
```

Comment:
Use `CodegenFallback`,as the sorting in codegen is quite complex, as we need to generate the java code to access the catalyst data type. (e.g. `Seq`)